src/components/data-table/column-sorting/column-sort.ts

```diff
@@ -0,0 +1,18 @@
+/*
```

Comment:
IMO, it should be a part of service file. If you separate type and interface in several files, it will never ending.